docs/design/contextual-tabs.md

```diff
@@ -0,0 +1,392 @@
+
+    - All the properties are required.
+    - The `id` property must be unique among all the groups in the tab. Use a brief, descriptive ID.
+    - The `label` is a user-friendly string to serve as the label of the group.
+    - The `icon` property's value is an array of objects that specify the icons that the group will have on the ribbon depending on the size of the ribbon and the Office application window.
+    - The `controls` property's value is an array of objects that specify the buttons and other controls in the group.
+
+    ```json
+    {
+        "id": "CustomGroup111",
+        "label": "Insertion",
+        "icon": [
+
+        ],
+        "controls": [
+
+        ]
+    }
+    ```
+
+1. Every group must have an icon of at least three sizes, 16x16 px, 32x32 px, and 80x80 px. Office decides which icon to use based on the size of the ribbon and Office application window. Add the following objects to the icon array. (If the window and ribbon sizes are large enough for at least one of the *controls* on the group to appear, then no group icon at all appears. For an example, watch the **Styles** group on the Word ribbon as you shrink and expand the Word window.) About this markup, note:
+
+    - Both the properties are required.
+    - The `size` property unit of measure is pixels. Icons are always square, so the number is both the height and the width.
+    - The `sourceLocation` property specifies the full URL to the icon.
+
+    > [!IMPORTANT]
+    > Just as you typically must change the URLs in a the add-in's manifest when you move from development to production (such as changing the domain from localhost to contoso.com), you must also change the URLs in your contextual tabs JSON.
+
+    ```json
+    {
+        "size": 16,
+        "sourceLocation": "https://cdn.contoso.com/addins/datainsertion/Images/Group16x16.png"
+    },
+    {
+        "size": 32,
+        "sourceLocation": "https://cdn.contoso.com/addins/datainsertion/Images/Group32x32.png"
+    },
+    {
+        "size": 80,
+        "sourceLocation": "https://cdn.contoso.com/addins/datainsertion/Images/Group80x80.png"
+    }
+    ```
+
+1. In the simple ongoing example, the group has only a single button. Add the following object as the only member of the `controls` array. About this markup, note:
+
+    - All the properties, except `enabled` are required.
+    - `type` specifies the type of control. The values can be "Button", "Menu", or "MobileButton".
+    - `id` can be up to 125 characters. 
+    - `actionId` must be the ID of an action defined in the `actions` array. (See above.)
+    - `label` is a user-friendly string to serve as the label of the button.
+    - `superTip` represents a rich form of tool tip. Both the `title` and `description` properties are required.
+    - `icon` specifies the icons for the button. The remarks above about the group icon apply here too.
```

Comment:
I think we're to avoid terms like "above" and "below" which may not make sense depending on the culture/language context
```suggestion
    - `icon` specifies the icons for the button. The previous remarks about the group icon apply here too.
```